services/gossip/adapter/tcp/peer_diff.go

```diff
@@ -0,0 +1,28 @@
+package tcp
+
+import "github.com/orbs-network/orbs-network-go/config"
+
+type gossipPeers map[string]config.GossipPeer
```

Comment:
this is a mapping from an orbs address to network endpoint. this is a little lost in the naming choice here, consider renaming `config.GossipPeer` to `config.GossipEndpoint` or `config.Gossip/NetworkAddress`